zipkin-cassandra/src/test/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactorySpec.scala

```diff
@@ -46,4 +48,32 @@ class CassandraSpanStoreFactorySpec extends FunSuite with Matchers {
       asList(new InetSocketAddress("1.1.1.1", 9143), new InetSocketAddress("2.2.2.2", 9042))
     )
   }
+
+  test("creatingClusterBuilder with SASL authentication null delimited utf8 bytes") {
+    TestFactory.nonExitingMain(Array(
+      "-zipkin.store.cassandra.user", "user",
+      "-zipkin.store.cassandra.password", "pass"
+    ))
+    val SASLhandshake = Array[Byte](0, 117, 115, 101, 114, 0, 112, 97, 115, 115)
```

Comment:
nit.. use the character form of 'u', 's', ... that way you don't have to explain what 117 is